### PR TITLE
Jmprieur/scopes

### PR DESCRIPTION
**Pull request by jmprieur**
- Fixing the controller of the Web app calls Web API test (adding verification of scopes)
- Adding a template for a Web API.

To test the template:

Build the NuGet package containing the .NET Core template

```Shell
cd ProjectTemplates
msbuild /t:pack
```

Install the template
```Shell
cd bin
cd Debug
dotnet new -i Microsoft.IdentityPlatform.Templates.0.1.0.nupkg
```

test the template
```Shell
mkdir test
cd test
dotnet new webapi2 --auth SingleOrg
msbuild 
```

you can also test the B2C `dotnet new webapi2 --auth IndividualB2C`

to uninstall the template, get back to `ProjectTemplates\bin\Debug` and run:

```Shell
dotnet new -u Microsoft.IdentityPlatform.Templates
```

### CODE REVIEW

File: ProjectTemplates/AspNetCoreMicrosoftIdentityPlatformTemplates.csproj [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageType>Template</PackageType>
+    <PackageVersion>0.1.0</PackageVersion>
+    <PackageId>Microsoft.IdentityPlatform.Templates</PackageId>
+    <Title>ASP.NET Core Web app and Web API templates with Microsoft identity platform</Title>
+    <Authors>Microsoft</Authors>
+    <Description>
+      Templates to use when creating a web app that signs in users or a protected Web API
```
Comment (jennyf19) [inline ProjectTemplates/AspNetCoreMicrosoftIdentityPlatformTemplates.csproj]: nit: one template or two templates? sincerely asking.

File: ProjectTemplates/AspNetCoreMicrosoftIdentityPlatformTemplates.csproj [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      with the Microsoft Identity platform (and call downstream APIs)</Description>
```
Comment (jennyf19) [inline ProjectTemplates/AspNetCoreMicrosoftIdentityPlatformTemplates.csproj]: lowercase i on Identity

**Comment by jmprieur** [comment]
Thanks for the feedback, @jennyf19 
I created https://github.com/AzureAD/microsoft-identity-web/issues/65 for `VerifyUserHasAnyAcceptedScope `

File: ProjectTemplates/AspNetCoreMicrosoftIdentityPlatformTemplates.csproj (hunk 1 above)
Comment (jmprieur) [reply ProjectTemplates/AspNetCoreMicrosoftIdentityPlatformTemplates.csproj]: the web app template will come soon :)
you are right for the moment, there is only one (there was 2 yesterday, but I remove an obsolete one)

**Pull request merged**